src/main/java/no/unit/nva/model/contexttypes/Degree.java

```diff
@@ -9,29 +11,31 @@
 @JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 public class Degree extends Book {
 
-    public Degree() {
-        super();
+    @JsonCreator
+    public Degree(@JsonProperty("series") BookSeries series,
+                  @JsonProperty("seriesTitle") String seriesTitle,
```

Comment:
same as in book.

Reply:
There is a replication of code here. Is there a possibility to not replicate the constructors or to make the hardcoded strings constants in the Book? Then we won't have to keep in sync all the field changes by hand

Reply:
This is a good point. Unfortunately, we need to express the constructor in this way, but I will migrate the strings to constants in Book.